iib/workers/tasks/build.py

```diff
@@ -868,13 +865,10 @@ def handle_add_request(
     )
     from_index_resolved = prebuild_info['from_index_resolved']
 
-    log.info('Checking if interacting with the legacy app registry is required')
-    legacy_support_packages = get_legacy_support_packages(
-        resolved_bundles, request_id, prebuild_info['ocp_version'], force_backport=force_backport
-    )
-    if legacy_support_packages:
-        validate_legacy_params_and_config(
-            legacy_support_packages, resolved_bundles, cnr_token, organization
+    if (cnr_token and organization) or force_backport:
+        log.warning(
```

Comment:
Can change this warning to `Legacy support is disabled in IIB. cnr_token, organization and force_backport will be ignored. `

Reply:
Since Jan is on PTO, I'll submit this change in a separate PR